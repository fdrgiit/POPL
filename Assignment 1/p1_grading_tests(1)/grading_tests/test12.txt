@#$%~

A 789.1REALX23.1INT

