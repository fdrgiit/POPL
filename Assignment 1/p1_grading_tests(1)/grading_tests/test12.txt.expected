1 38 ERROR
1 38 ERROR
1 38 ERROR
1 38 ERROR
1 38 ERROR
3 37 A 
3 34 789 
3 33 .
3 34 1 
3 35 REALX23.1INT 
